force-app/main/default/lwc/reviewSessions/reviewSessions.js

```diff
@@ -52,6 +54,15 @@ export default class ReviewSessions extends LightningElement {
 
         if (!this._serviceSessions.length) {
             this.getSessions();
+        } else {
+            // This is fired when the back button is clicked
+            // eslint-disable-next-line @lwc/lwc/no-async-operation
+            setTimeout(
+                function() {
+                    this.handleDispatchLoadedEvent();
+                }.bind(this),
+                2000
```

Comment:
Nice work on setting the timeouts, I think we really only need to display a spinner if it is taking awhile to load the screen. I do like the screens are consistent while transitioning but I think it might be more user friendly to just load the page right away when possible.